### ISSUE TITLE
Flaky Firefox WebRTC test

### ISSUE THREAD

**Issue by achingbrain** [libp2p/js-libp2p#2642]
Sometimes the receiving end of the `can send a large file` WebRTC test doesn't receive all of the bytes when running in Firefox, Chrome is fine:

```
  1) basics
       can send a large file:

      expected Uint8Array[ 0, 0, 0, 0, 0, 0, 0, …(5242845) ] to equal Uint8Array[ 0, 0, 0, 0, 0, 0, 0, …(5242881) ]
      + expected - actual
```

This fails in CI quite often and can be reproduced locally too.

It may be related to https://bugzilla.mozilla.org/show_bug.cgi?id=1902850 - the fix for this should ship in Firefox 129 on August 6tth.

**Comment by achingbrain** [comment]
> It seems to happen when there are several datachannels open on the connection simultaneously

I don't think this is the case, it still happens if I add a delay in the test to ensure there are no other channels running.

Investigation continues.

**Comment by SgtPooki** [comment]
looks like playwright has the latest Firefox set, but hasn't released an updated version to NPM yet: https://github.com/microsoft/playwright/commit/81a1253b10cc6f5d2709d3ea700ed3cafbae5f89

**Comment by tabcat** [comment]
This looks to be resolved as I cannot get the test to fail in Firefox. The Firefox version used by playwright-core is 131. https://bugzilla.mozilla.org/show_bug.cgi?id=1902850 changes have been present since 129.

**Comment by achingbrain** [comment]
CI is still really flaky around these tests 😩
